Java/src/1-closure.java

```diff
@@ -0,0 +1,19 @@
+import java.util.function.*;
+
+class Closure {
+
+    public static void main(String[] args)  {
+        Function<Double,Function<Double,Double>> createLog = (Double base) -> (Double n) -> log(base, n);
```

Comment:
Code style. And could you split this line to fit 100 symbols in line, please? Just a code style suggestion.